### PR TITLE
Add manifest for io.github.rafaelfassi.QLogExplorer

### PR DESCRIPTION
**Pull request by rafaelfassi**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by rafaelfassi** [comment]
bot, build qlogexplorer

**Comment by rafaelfassi** [comment]
bot, build io.github.rafaelfassi.QLogExplorer

File: io.github.rafaelfassi.QLogExplorer.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+app-id: io.github.rafaelfassi.QLogExplorer
+runtime: org.kde.Platform
+runtime-version: '5.15'
```
Comment (hfiguiere) [inline io.github.rafaelfassi.QLogExplorer.yaml:3]:
```suggestion
runtime-version: '5.15-21.08'
```

File: io.github.rafaelfassi.QLogExplorer.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+command: qlogexplorer
+finish-args:
+  # Required for Qt render.
+  - --socket=wayland
+  - --device=dri
+  # Necessary if X11 fallback takes place.
+  - --socket=fallback-x11
+  - --share=ipc
+  # Necessary to open system log files.
+  - --filesystem=host
```
Comment (hfiguiere) [inline io.github.rafaelfassi.QLogExplorer.yaml:17]: are you sure you can view them?

File: io.github.rafaelfassi.QLogExplorer.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  # Local sockets are used to detect other running instance and forward any open file request to it.
+  # Qt requires Network module to use sockets. Even though an error message is print on app startup
+  # when no network permission is granted, it's working properly.
+  # - --share=network
+modules:
+  - name: qlogexplorer
+    buildsystem: simple
```
Comment (hfiguiere) [inline io.github.rafaelfassi.QLogExplorer.yaml:24]: you should use `buildsystem: cmake-ninja` and then add extra stuff to the `post-install`

**Review by hfiguiere — CHANGES_REQUESTED** [review]
also the app data misses `releases` and `content_rating`

File: io.github.rafaelfassi.QLogExplorer.yaml (hunk 2 above)
Comment (rafaelfassi) [reply io.github.rafaelfassi.QLogExplorer.yaml:17]: Yeah, the user can open whatever file he has read permission.

File: io.github.rafaelfassi.QLogExplorer.yaml (hunk 3 above)
Comment (rafaelfassi) [reply io.github.rafaelfassi.QLogExplorer.yaml]: Thanks for pointing that. Done!

File: io.github.rafaelfassi.QLogExplorer.yaml (hunk 1 above)
Comment (rafaelfassi) [reply io.github.rafaelfassi.QLogExplorer.yaml]: Thanks for the suggestion.
It's applied now.

**Comment by rafaelfassi** [comment]
bot, build io.github.rafaelfassi.QLogExplorer

**Comment by rafaelfassi** [comment]
bot, build io.github.rafaelfassi.QLogExplorer

File: io.github.rafaelfassi.QLogExplorer.yaml (hunk 2 above)
Comment (barthalion) [reply io.github.rafaelfassi.QLogExplorer.yaml:17]: If this is about accessing system logs, I would change it to `--filesystem=host` and `--filesystem=/var/log`.

**Comment by rafaelfassi** [comment]
Tested and working properly :slightly_smiling_face:

**Comment by rafaelfassi** [comment]
Looks like this pull request is kind of stopped.
Is there something missing from my side?

**Comment by hfiguiere** [comment]
patience

**Pull request closed without merging**